src/card.py

```diff
@@ -1,10 +1,10 @@
-import json
-#Implementation of a standard Poker card, with ability to output to JSON
-class Card:
-    def __init__(self, rank, suit):
-        self.rank = {"rank": rank}
-        self.suit = {"suit": suit}
+from typing import NamedTuple
 
-    def to_json(self):
-         return json.dumps({"rank": self.rank, 
-                            "suit": self.suit})
\ No newline at end of file
+
+class Card(NamedTuple):
```

Comment:
What's the advantage in using a NamedTuple over a normal class?

Reply:
@semmypurewal mostly the built-in `_todict` method and immutability.

Reply:
Ah, right! I forgot about that.